internal/provider/burn_alert_resource_test.go

```diff
@@ -878,6 +921,43 @@ func burnAlertAccTestSetup(t *testing.T) (string, string) {
 	return dataset, slo.ID
 }
 
+func getNewDatasetAndSLO(t *testing.T) (string, string) {
+	t.Helper()
+
+	ctx := context.Background()
+	c := testAccClient(t)
+
+	dataset, err := c.Datasets.Create(ctx, &client.Dataset{
+		Name: acctest.RandomWithPrefix("test"),
```

Comment:
I'd rather see this version as the sweepers are cleaning things up that start with `test.` (this variation is making things like `test-3280250866423422428`)
```suggestion
		Name: test.RandomStringWithPrefix("test.", 10),
```